src/core/themePreview.js

```diff
@@ -11,7 +11,9 @@ export default function themeAction(node, action, _doc = document) {
   node.dispatchEvent(event);
 }
 
-export function getThemeData({ id, name, headerURL, footerURL, textcolor, accentcolor, author }) {
+export function getThemeData({
+id, name, description, headerURL, footerURL, textcolor, accentcolor, author,
```

Comment:
Oh, sorry, can you just indent this? Sorry the linter didn't catch it but that's our usual style for this kind of argument setup.